lib/ansible/modules/cloud/scaleway/scaleway_compute.py

```diff
@@ -643,6 +654,7 @@ def main():
         region=dict(required=True, choices=SCALEWAY_LOCATION.keys()),
         commercial_type=dict(required=True),
         enable_ipv6=dict(default=False, type="bool"),
+        boot_type=dict(default="bootscript", choices=['bootscript', 'local']),
```

Comment:
I think the default should be local @CedricCabessa Could you confirm?

Reply:
see my remark above https://github.com/ansible/ansible/pull/55582/files/59c73d2e165688f59e2d05db784bc1f950708fb4#r277599212